assembl/static2/package.json

```diff
@@ -9,8 +9,8 @@
   "scripts": {
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "dev": "webpack -d --watch",
-    "build": "webpack -p",
+    "dev": "webpack -d --watch --progress --colors",
```

Comment:
I have --progress and --colors in my webpack version 1.14. The --progress flag is documented in http://webpack.github.io/docs/cli.html#progress . While you mention it, we should take care of the effect of these options in our build logs, and remove them if it creates information loss. But on my side for now I did not see any.